apollo-common/src/main/java/com/ctrip/framework/apollo/common/controller/GlobalDefaultExceptionHandler.java

```diff
@@ -153,4 +156,44 @@ private void printLog(String message, Throwable ex, Level logLevel) {
     Tracer.logError(ex);
   }
 
+  /**
+   * 获取请求参数
+   * @param request
+   * @return
+   */
+  private Map getInputParams(HttpServletRequest request) {
+    Map<String, String> map = new HashMap();
+    Enumeration paramNames = request.getParameterNames();
+    while (paramNames.hasMoreElements()) {
+      String paramName = (String) paramNames.nextElement();
+      String[] paramValues = request.getParameterValues(paramName);
+      if (paramValues.length == 1) {
```

Comment:
how about paramValues.length > 1?

Reply:
if:   **paramValues < 1**

in this case, the paramValue is meaningless, for a practical application scenario like Apollo。At the same time, the simplicity of post-event log audit is considered.

Reply:
I think http supports one param with multiple values, such as `http://xxx.com?q=a&q=b`, though it's not a typical use case for apollo.

Since we want to expose detailed information in logs, I think it's better to log everything we have, e.g. for the above  case, log `q=a,b`.

Another minor point is ServletRequest has a method `getParameterMap` which may be easier to use since we are printing all the params.